pkgs/development/libraries/libosinfo/default.nix

```diff
@@ -19,21 +19,6 @@ stdenv.mkDerivation rec {
   buildInputs = [ glib libsoup libxml2 libxslt ];
   checkInputs = [ check curl perl ];
 
-  patches = [
-    ./osinfo-db-data-dir.patch
```

Comment:
You accidentally removed our patch for fixing paths.

Could you change it to 

```nix
(substituteAll {
  src = ./osinfo-db-data-dir.patch;
  OSINFO_DB_DATA_DIR = "${osinfo-db}/share";
})
```

so we could drop the `postPatch` section?

Reply:
that's my bad :(

Reply:
added back :)

Reply:
i was able to remove the patchShebangs altogether, the package doesn't export any scripts that i can see, and still builds without the help of a manual patchShebang call.

Reply:
used the substituteAll, seems cleaner :)